tests/linalg_test.py

```diff
@@ -1016,7 +1016,8 @@ def tensor_maker():
       ((1, 1), (1, 1)),
       ((4, 4), (4,)),
       ((8, 8), (8, 4)),
-      ((1, 2, 2), (3, 2)),
+      # This case errors with NumPy 2.0 solve semantics.
```

Comment:
What action do you intend? Remove the test when we drop NumPy 1? Fix the test?

Reply:
Hmm, good point. Maybe we should just remove it altogether?

Reply:
Done.